driverless/driverless/src/main/java/com/upgrad/hirewheels/validator/BookingValidatorImpl.java

```diff
@@ -25,19 +25,19 @@ public void validateBooking(AddBookingDTO vehicle) throws ParseException {
         String formatDropOffDate = sdf.format(vehicle.getDropoffDate());
         Date parsedDropOffDate = sdf.parse(formatDropOffDate);
```

Comment:
Please check if id=0 catches the error where no value is passed corresponding to it in the request body. If this works, please change the error messages to the previous one. For ex,
"Not a Valid UserId Id should be changed" to "UserId Id cannot be empty or null"